kubernetes-manifests/lanebreach-backend.deployment.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: node-lanebreach-backend
+  labels:
+    app: node-lanebreach
+    tier: backend
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: node-lanebreach
+        tier: backend
+    spec:
+      containers:
+      - name: backend
+        image: node-lanebreach-backend
+        imagePullPolicy: IfNotPresent
+        ports:
+        - name: http-server
+          containerPort: 3000
+        env:
+        - name: PORT
+          value: "3000"
+        - name: RANDOM_ENV_VAR
```

Comment:
I had some issue where the docker container wasn't being pulled (due to a silent network issue on my laptop) and thus wasn't picking up the env. I added one here to see if it would show up in printenv... it did... I was shocked... then realized what was going on. Related to that other accidental inclusion with the ImagePullPolicy. Will fix.